static/app/components/workflowEngine/simpleTable/index.tsx

```diff
@@ -0,0 +1,101 @@
+import {type CSSProperties, useCallback, useRef} from 'react';
+import {useResizeObserver} from '@react-aria/utils';
+
+import {
+  Body as Panel,
+  Grid,
+  GridBody,
+  GridBodyCell,
+  GridBodyCellStatus,
+  GridHead,
+  GridHeadCellStatic,
+  GridRow,
+} from 'sentry/components/gridEditable/styles';
+
+export interface ColumnConfig<Cell> {
+  Header: () => React.ReactNode;
+  Cell?: (props: {value: Cell}) => React.ReactNode;
+  /** @default 1fr */
+  width?: CSSProperties['gridTemplateColumns'];
```

Comment:
Could we possibly get an example in the stories with custom widths?

Reply:
added in dfe0fb006ec